db/host_id.hh

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Why in `db/`?

Reply:
I wouldn't like to add more files to the root
so I adopted the first occurrence that was in db::config.
Maybe locator/host_id makes more sense

Reply:
db/ is typically for replica code (though we're re-organizing it in replica/). Agree locator/ is better.